src/app/template-list/template-list.component.ts

```diff
@@ -64,8 +64,18 @@ export class TemplateListComponent implements OnInit {
       this.router.navigate(['template',1])
     }
 
-    
-    
+    removeTemplate(tmpl): void {
+      this.templatesService.currTmpl = tmpl;
+      this.templatesService
+      .removeTemplate(tmpl).
```

Comment:
tmpl is an object that contains a template_id and name. so in templatesService.removeTemplate() we should pass in tmpl.template_id as the parameter